### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by and-lucas-kelly**
updated the rails call to include bin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ The migrations are stored in the `db/data` folder.
 
 - To generate a migration: `rails g data_migration add_this_to_that`
```
Comment (WorkSutton) [inline README.md:192]: @and-lucas-kelly being really pedantic here, but I think for extra clarity it should say `data migration`.
I know its in the topic area, but I have added this as on the line directly after we call it a `data migration`

Comment (and-lucas-kelly) [reply README.md:192]: makes sense to me.

**Review by nomasprime — CHANGES_REQUESTED** [review]
Been thinking about this and don't think it's really worth our time:

1. Can be run in multiple ways
2. Not sure runtime paths need to be in scope of this doc
3. If it is, should be updated with multiple ways and throughout, eg, bullet above is missing bin

**Comment by and-lucas-kelly** [comment]
after review from RJ and MS decided changes not needed or needs an overall review and update.

**Pull request closed without merging**